doc/contributing/collaborator-guide.md

```diff
@@ -392,13 +392,21 @@ metadata. Raise a pull request like any other change.
 
 Treat commits that introduce new core modules with extra care.
 
-Check if the module's name conflicts with an existing ecosystem module. If it
-does, choose a different name unless the module owner has agreed in writing to
-transfer it.
+New modules must only be added with the `node:` prefix.
 
-If the new module name is free, register a placeholder in the module registry as
-soon as possible. Link to the pull request that introduces the new core module
-in the placeholder's `README`.
+When adding promises to an existing API, add `/promises`
+(`inspector/promises`, etc.). Apply the `semver-major` label to the addition.
```

Comment:
if node added a semver-major change that prevented any slashed requires/imports from core module names, then this category would no longer be semver-major after that.

Reply:
I created https://github.com/nodejs/node/issues/44651 for that.